src/k8-client/src/cert.rs

```diff
@@ -179,14 +179,23 @@ where
 
             let token_output = Command::new(exec.command.clone())
                 .args(exec.args.clone())
-                .output()?;
+                .output()
+                .map_err(|e| {
+                    let msg = format!("executing {} {:#?}", exec.command, exec.args);
+                    info!(target: "k8_client", "error {msg}");
```

Comment:
shouldn't this be a `debug!`?

Reply:
Oh I see it is an error. Maybe `warn` or `error` instead?

Reply:
I think it should be `debug!` like the others, since the error is bubbled to the caller.

Reply:
You and Nick had the same comment.  I'll update to `debug!`.  `error!` generates a user visible log line at the default fluvio cli settings that I don't think is desirable.

Reply:
or remove it,  otherwise it will leads to duplicate trace

Reply:
would it be a duplicate of https://github.com/infinyon/fluvio/blob/ed94fb2e2beb8da70d898c71b329ff197e55afef/crates/fluvio-cluster/src/start/k8.rs#L589 ?

Reply:
Not exactly, without these extra traces, you get an pretty generic io error where you can't differentiate what happened on these paths.  I could remove the traces, and map_err to add more customized context inside a IoError, and then the parent code in the link would work fine.